extensions/binding/org.eclipse.smarthome.binding.hue/README.md

```diff
@@ -39,39 +51,39 @@ Bridge hue:bridge:1 [ ipAddress="192.168.0.64", userName="qwertzuiopasdfghjklyxc
 The bulbs are identified by the number that the Hue bridge assigns to them (also shown in the Hue app as an identifier).
 Thus, all if needs for manual configuration is this single value like
 ```
-LCT001 bulb1 [ lightId="1" ]
+0210 bulb1 [ lightId="1" ]
 ```
 
 ## Channels
 
 All devices support some of the following channels:
 
-| Channel Type ID | Item Type    | Description  |
-|-----------------|------------------------|--------------|----------------- |------------- |
-| color | Color       | This channel supports full color control with hue, saturation and brightness values. |
-| brightness | Dimmer       | This channel supports adjusting the brightness value. Note that this is not available, if the color channel is supported. |
-| color_temperature | Dimmer       | This channel supports adjusting the color temperature from cold (0%) to warm (100%) |
-
+| Channel Type ID | Item Type    | Description  | Thing types supporting this channel | 
+|-----------------|------------------------|--------------|----------------- |------------- |---------|
+| color | Color       | This channel supports full color control with hue, saturation and brightness values. | 0200, 0210 | 
+| brightness | Dimmer       | This channel supports adjusting the brightness value. Note that this is not available, if the color channel is supported. | 0100, 0200, 0210, 0220 | 
+| color_temperature | Dimmer       | This channel supports adjusting the color temperature from cold (0%) to warm (100%) | 0210, 0220 |
+| alert | Option | This channel supports displaying alerts by flashing the bulb either once opr multiple times. Valid options are: None, Alert, Long Alert | 0000, 0100, 0200, 0210, 0220 |
```

Comment:
Item type "Option"...? This is new to me...
And "Long Alert" is a valid command for it? I thought it is NONE, SELECT and LSELECT.